.mailmap

```diff
@@ -22,6 +22,7 @@ Lee A. Taylor <taylor16@llnl.gov>               Lee A. Taylor <taylor@cab688.lln
 Lee A. Taylor <taylor16@llnl.gov>               Lee A. Taylor <taylor@cab688.llnl.gov>
 Lee A. Taylor <taylor16@llnl.gov>               Lee Taylor <ltaylor@llnl.gov>
 Lee A. Taylor <taylor16@llnl.gov>               Lee Taylor <taylor16@llnl.gov>
+Matt Larsen <larsen30@llnl.gov>		        mclarsen <larsen30@llnl.gov>
```

Comment:
Why is the format of the new entry in the right column different than all the others?

Reply:
Hmm...I am not sure why it is rendered like that. When I "cat" the file, edit in "vim" or "emacs" the indentation appears to be correct.

Reply:
I was also wondering about "mclarsen" vs. "Matt Larsen".